plugins/woocommerce-admin/client/marketing/overview-multichannel/Channels/RecommendedChannelsList.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import { Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { CardDivider } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { SmartPluginCardBody } from '~/marketing/components';
+import { RecommendedChannel } from '~/marketing/data-multichannel/types';
+import './Channels.scss';
+
+type RecommendedChannelListPropsType = {
+	recommendedChannels: Array< RecommendedChannel >;
+	onInstalledAndActivated?: () => void;
+};
+
+export const RecommendedChannelsList: React.FC<
```

Comment:
💅  I found a bit confusing this name for the component. We have also `RecommendedChannels` as a wrapper. I would name it maybe as `RecommendedChannelsPluginList` or as an alternative rename `RecommendedChannels` as `RecommendedChannelsCard`?

Reply:
I have made some changes and removed `RecommendedChannels` and `RecommendedChannelsList`, essentially collapsing the React component tree, in [`0eb318c` (#36541)](https://github.com/woocommerce/woocommerce/pull/36541/commits/0eb318c1f5153844354d82515e57385fcd1a35fc).